qiita_pet/handlers/study_handlers/sample_template.py

```diff
@@ -277,9 +283,19 @@ def post(self):
         study_id = int(self.get_argument('study_id'))
         filepath = self.get_argument('filepath')
         data_type = self.get_argument('data_type')
+        direct_upload = self.get_argument('data_type', False)
```

Comment:
Is this correct? This line is querying for `data_type` but is assigning a boolean value *and* the previous line is also querying for `data_type`.